### PR TITLE
Integrate operator with CVO

### PR DESCRIPTION
**Pull request by ironcladlou**

### CODE REVIEW

File: images/cluster-ingress-operator/Dockerfile [hunk 1]
```diff
@@ -22,3 +22,5 @@ USER 1001
 LABEL io.k8s.display-name="OpenShift cluster-ingress-operator" \
       io.k8s.description="This is a component of OpenShift Container Platform and manages the lifecycle of cluster ingress components." \
       maintainer="Dan Mace <dmace@redhat.com>"
+
+LABEL io.openshift.release.operator true
```
Comment (pravisankar) [inline images/cluster-ingress-operator/Dockerfile]: Will this enable CVO to install cluster-ingress-operator as part of the cluster deployment?
If yes, don't we want some basic CI gating tests for new prs that validates interaction with CVO?

Comment (ironcladlou) [reply images/cluster-ingress-operator/Dockerfile]: Yeah, I'm doing testing now and will probably have to make a PR to fix some RBAC stuff

File: pkg/manifests/manifests_test.go [hunk 2]
```diff
@@ -29,30 +29,6 @@ func TestManifests(t *testing.T) {
 		},
 	}
 
-	if _, err := f.OperatorNamespace(); err != nil {
-		t.Fatal(err)
-	}
-	if _, err := f.OperatorCustomResourceDefinition(); err != nil {
-		t.Fatal(err)
-	}
-	if _, err := f.OperatorRole(); err != nil {
```
Comment (pravisankar) [inline pkg/manifests/manifests_test.go]: Looks like we can keep f.OperatorRole() and f.OperatorRoleBinding()?

Comment (ironcladlou) [reply pkg/manifests/manifests_test.go]: We need to revisit RBAC holistically after this

**Pull request merged**